### ISSUE TITLE
hovering over a bookmark bar item shows corners as rounded 

### ISSUE THREAD

**Issue by LaurenWags** [brave/brave-browser#7210]
## Description 
On C79 builds when you hover over a bookmark bar item the highlighted item is rounded instead of more square. C78 builds show the hover correctly.

## Steps to Reproduce

   1. Add bookmark/folder to bookmarks bar.
   2. Hover over item.
   3. 

## Actual result:
See that highlighted item corners are rounded:
<img width="1194" alt="Screen Shot 2019-12-04 at 3 20 01 PM" src="https://user-images.githubusercontent.com/28145373/70178392-35bc3680-16aa-11ea-9a8c-d273dab1016a.png">

## Expected result:
Corners should be more square
<img width="1195" alt="Screen Shot 2019-12-04 at 3 18 56 PM" src="https://user-images.githubusercontent.com/28145373/70178403-3a80ea80-16aa-11ea-8026-7307ae14af98.png">

## Reproduces how often: 
easily

## Brave version (brave://version info)

Brave | 1.1.18 Chromium: 79.0.3945.56 (Official Build) (64-bit)
-- | --
Revision | 73cc6bf591f792b99f8fc7cdfb8addedbd084bf8-refs/branch-heads/3945@{#788}
OS | macOS Version 10.13.6 (Build 17G5019)

Brave | 1.4.3 Chromium: 79.0.3945.56 (Official Build) nightly (64-bit)
-- | --
Revision | 73cc6bf591f792b99f8fc7cdfb8addedbd084bf8-refs/branch-heads/3945@{#788}
OS | macOS Version 10.13.6 (Build 17G5019)

## Version/Channel Information:

- Can you reproduce this issue with the current release? not with 1.0.1
- Can you reproduce this issue with the beta channel? unsure
- Can you reproduce this issue with the dev channel? unsure
- Can you reproduce this issue with the nightly channel? yes with 1.4.3

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? n/a
- Does the issue resolve itself when disabling Brave Rewards? n/a
- Is the issue reproducible on the latest version of Chrome? n/a

## Miscellaneous Information:
cc @bsclifton @rebron @brave/legacy_qa

**Comment by LaurenWags** [comment]
Does not reproduce on 1.1.17 (last C78 1.1.x build):
<img width="1194" alt="Screen Shot 2019-12-04 at 3 22 03 PM" src="https://user-images.githubusercontent.com/28145373/70179096-6d77ae00-16ab-11ea-943e-7d43ddb5dd2b.png">

**Comment by LaurenWags** [comment]
@bsclifton noticed that the separator isn't always shown and that drag/drop of bookmarks into folders is difficult. When checking this issue we should retest this part as well.

**Comment by GeetaSarvadnya** [comment]
Reproduced in Windows 10 x64 - 1.1.18

**Comment by LaurenWags** [comment]
Adding 1.1.x milestone as this is likely a C79 regression we want resolved before releasing.

**Comment by kjozwiak** [comment]
@bsclifton @rebron are we planning on getting this into the next release? It looks like https://github.com/brave/brave-core/pull/4135 has been completed/reviewed. We should get this merged into `Nightly`/QA'd and get the uplift PR's going if this is something we want out in `1.1.x`.

**Comment by bsclifton** [comment]
Waiting on code review from @bridiver https://github.com/brave/brave-core/pull/4135

**Comment by LaurenWags** [comment]
Verified passed with

Brave | 1.1.20 Chromium: 79.0.3945.74 (Official Build) (64-bit)
-- | --
Revision | 99b6d241cf84f7fa7434ae8a59f9e74eb1464280-refs/branch-heads/3945@{#888}
OS | macOS Version 10.13.6 (Build 17G5019)

* Verified STR from description
<img width="261" alt="Screen Shot 2019-12-10 at 8 57 14 AM" src="https://user-images.githubusercontent.com/28145373/70535580-51509280-1b2b-11ea-8cee-45b933ad4104.png">
<img width="257" alt="Screen Shot 2019-12-10 at 8 57 22 AM" src="https://user-images.githubusercontent.com/28145373/70535585-531a5600-1b2b-11ea-8199-2a3343434032.png">
* Verified able to drag and drop items on bookmark bar without issue.
